color-picker-project/src/components/colorpicker/ColorPicker.css

```diff
@@ -19,10 +19,11 @@
 
 .color-picker-color {
     margin-bottom: 3rem;
+    width: '70%'
 }
 
 .color-picker-material {
-    width: 200px;
+    width: 4rem;
```

Comment:
4rem is only 64px.  In the jsx file width is also set for this class inline, at 150px. I'd suggest setting width to 150px here in the css, and removing the inline style from the jsx file (I'll make a separate comment on that line too).

Reply:
changed.